extensions/amp-access/0.1/jwt.js

```diff
@@ -29,6 +31,22 @@ import {tryParseJson} from '../../../src/json';
  */
 let JwtTokenInternalDef;
 
+/**
+ * Converts a text in PEM format into a binary array buffer.
+ * @param {string} pem
+ * @return {!Uint8Array}
+ * @visibleForTesting
+ */
+export function pemToBytes(pem) {
+  // Remove pem prefix, e.g. "----BEGIN PUBLIC KEY----".
+  // Remove pem suffix, e.g. "----END PUBLIC KEY----".
+  // Remove surrounding whitespace.
+  // Remove line breaks.
+  const removeRegex = /^\s*-+BEGIN[^-]*-+\s*|\s*-+END[^-]*-+\s*$|[\r\n]/g;
```

Comment:
This isn't an equivalent function, because the `\s` is now tied to the prefix and suffix.

Reply:
That change was to incorporate the first `trim()`, but I've reverted it